src/cli/commands/install.js

```diff
@@ -660,7 +660,7 @@ export class Install {
     }
 
     // build lockfile location
-    const loc = path.join(this.config.cwd, constants.LOCKFILE_FILENAME);
+    const loc = path.join(this.config.worktreeFolder || this.config.cwd, constants.LOCKFILE_FILENAME);
```

Comment:
The `this.config.worktreeFolder || this.config.cwd` expression is used in multiple places.
How about moving it to a function that would be self explaining?
Should we make `this.config.cwd` private?